packages/node-cli/src/core/CommandOutputHandler.js

```diff
@@ -6,9 +6,11 @@
 
 const NodeUtils = require('./../utils/NodeUtils');
 const { unwrapExceptionMessage, unwrapInformationMessage } = require('./../utils/ExceptionUtils');
+const ActionResult = require('../commands/actionresult/ActionResult');
 
 module.exports = class CommandOutputHandler {
 	showSuccessResult(actionResult, formatOutputFunction) {
+		// TODO assert(actionResult instanceof ActionResult);
```

Comment:
Since all the results are build now with ActionResult.Builder, I think this assert isn't neccessary. Also, you don't need the require() in line 9